docker/Dockerfile

```diff
@@ -3,6 +3,8 @@
 
 FROM ubuntu:20.04
 
+RUN set -e
```

Comment:
I hate to be sprinkling this link around everywhere :) but just in case my hunch is right, I need to leave it here too haha: https://github.com/ava-labs/bitcoin-sgx-bridge/pull/263#discussion_r1376351645

Reply:
See my comment on your other comment :)